docs/api/covid_hosp_facility_lookup.md

```diff
@@ -54,8 +54,16 @@ supported.
 | --- | --- | --- |
 | `result` | result code: 1 = success, 2 = too many results, -2 = no results | integer |
 | `epidata` | list of results | array of objects |
-| `epidata[].hospital_pk` | facility identified by this row | string |
-| `epidata[].*` | see the [data dictionary](https://healthdata.gov/covid-19-reported-patient-impact-and-hospital-capacity-facility-data-dictionary) |  |
+| `epidata[].hospital_pk` | unique identifier for this facility (will match CCN if CCN exists) | string |
+| `epidata[].state` | two-letter state code | string |
+| `epidata[].ccn` | CMS Certification Number for this facility | string |
+| `epidata[].hospital_name` |  | string |
+| `epidata[].address` |  | string |
+| `epidata[].city` |  | string |
```

Comment:
Super minor aesthetic thing: could fill in [blanks](https://github.com/cmu-delphi/delphi-epidata/blob/4fff03c83f1c4a60876043ededa314b3f81285c3/docs/api/covid_hosp_facility_lookup.md#response) with "facility name", "facility address", "facility city".